### DISCUSSION TITLE
parse time with locale

### DISCUSSION THREAD

**Discussion by void-mian** [abseil/abseil-cpp#1574] [Q&A]
I found that absl::parseTime uses cctz::detail::parseTM which is locale specific. Is this by design?

**Comment by derekmauro** [comment]
This is a question for @devbww or https://github.com/google/cctz.

**Comment by devbww** [comment]
> I found that absl::parseTime uses cctz::detail::parseTM which is locale specific. Is this by design?

Yes.  Apart from its special extensions, `absl::ParseTime()` uses `strptime()` or `std::get_time()` to implement its format specifiers, many of which are defined as locale-dependent.  Maintaining this property is indeed by design.
